components/capture-build-scan-url-maven-extension/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -2,9 +2,9 @@
   <components>
     <component>
       <role>com.gradle.maven.extension.api.GradleEnterpriseListener</role>
-      <role-hint>capture-build-scan-url</role-hint>
-      <implementation>com.gradle.CaptureBuildScansListener</implementation>
-      <description>Captures the build scan URL from Maven builds</description>
+      <role-hint>configure-gradle-enterprise</role-hint>
```

Comment:
Should we also rename the Maven extension project from `capture-build-scan-url-maven-extension` to `configure-gradle-enterprise-maven-extension`? We would also need to update the name of the jar on the classpath when invoking Maven. Without it, this renaming feels only half complete, but I am also fine leaving it.

Reply:
+1. but let's do after the merge to avoid too many changes. (for the list of 2.3.2)